app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,29 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+
+  before_action :authenticate_user!, only: %i[create index]
+
+  def create
+    rent = current_user.rent.build(create_params)
```

Comment:
I'm pretty sure this should be `current_user.rents.build(create_params)` with `rent` in plural!

Reply:
:thinking: it doesn't work
NoMethodError: undefined method `rents' for #<User:0x00005639ccec9008> 
Did you mean?  rent rent=

Reply:
We have a bug here https://github.com/wolox-training/lv-rails/blob/4b744c088381c8e0c6504ee625dbc2640fd08d27/app/models/user.rb#L12
It should be `:rents` in plural, as it is a collection of rents, not one!

Reply:
:sweat_smile: i thought has_many could automatic pluralize